app/controllers/parks_controller.rb

```diff
@@ -48,6 +48,14 @@ def destroy
     end
   end
 
+  def search
+    if params[:search].nil?
+      @parks = []
+    else
+      @parks = Park.where("title ILIKE ?", "%#{params[:search]}%")
+    end
+  end
```

Comment:
this actually should be part of the `index` action. Your `index` should display all parks if there is no `params[:search]`. It's up to you what to do when `params[:search] is an empty string`, you could display nothing as you are doing now. If there is `params[:search]` then you should search.